features/fetch-crl/config.pan

```diff
@@ -16,25 +16,41 @@ include { 'features/fetch-crl/rpms' + RPMS_CONFIG_SUFFIX };
 # Define fetch-crl version if not defined when adding RPMs
 variable FETCH_CRL_VERSION ?= '3.0';
 
-# ----------------------------------------------------------------------------
+# ---------------------------------------------------------------------------- 
 # fetch-crl configuration
-# ----------------------------------------------------------------------------
-include { 'components/sysconfig/config' };
-"/software/components/sysconfig/files/fetch-crl/CRLDIR" = SITE_DEF_CERTDIR;
-"/software/components/sysconfig/files/fetch-crl/FORCE_OVERWRITE" = if ( FETCH_CRL_FORCE_OVERWRITE ) {
-                                                                     'yes';
-                                                                   } else {
-                                                                     'no';
-                                                                   };
-"/software/components/sysconfig/files/fetch-crl/QUIET" = if ( FETCH_CRL_QUIET ) {
-                                                           'yes';
-                                                         } else {
-                                                           'no';
-                                                         };
-
-# ----------------------------------------------------------------------------
+# ---------------------------------------------------------------------------- 
+include { 'components/metaconfig/config' };
+# property daemons is not supported in older version of metaconfig but unfortunately
+# metaconfig doesn't publish its version until 15.12. Until the version is defined
+# use QUATTOR_RELEASE to determine the version.
+'/software/components/metaconfig/services/{/etc/sysconfig/fetch-crl}/daemons' = {
+  if ( QUATTOR_RELEASE >= '15' ) {
+    nlist('fetch-crl-boot', 'restart');
+  } else {
+    null;
+  };
+};
+'/software/components/metaconfig/services/{/etc/sysconfig/fetch-crl}/backup' = '.old';
+'/software/components/metaconfig/services/{/etc/sysconfig/fetch-crl}/module' = 'tiny';
+'/software/components/metaconfig/services/{/etc/sysconfig/fetch-crl}/contents' = {
+  SELF["CRLDIR"] = SITE_DEF_CERTDIR;
+  SELF["FORCE_OVERWRITE"] = if ( FETCH_CRL_FORCE_OVERWRITE ) {
```

Comment:
`TextRender` has all the code to autoconvert booleans in e.g. `yes` or `no`, but needs this exposed via metaconfig (i tought i already did this ...). this code would thena become
```pan
prefix '/software/components/metaconfig/services/{/etc/sysconfig/fetch-crl}/';
'options/yesno' = true;
prefix '/software/components/metaconfig/services/{/etc/sysconfig/fetch-crl}/contents';
'FORCE_OVERWRITE' = FETCH_CRL_FORCE_OVERWRITE;
```